alibi_detect/saving/loading.py

```diff
@@ -235,25 +240,22 @@ def _load_preprocess_config(cfg: dict,
     return partial(preprocess_fn, **kwargs)
 
 
-def _load_model_config(cfg: dict,
-                       backend: str) -> Callable:
+def _load_model_config(cfg: dict) -> Callable:
     """
-    Loads TensorFlow, PyTorch and scikit-learn models (currently only TensorFlow supported), from a model config
-    dict.
+    Loads supported models from a model config dict.
 
     Parameters
     ----------
     cfg
         Model config dict. (see pydantic model schemas).
-    backend
-        The backend.
 
     Returns
     -------
     The loaded model.
     """
 
     # Load model
+    flavour = cfg['flavour']
```

Comment:
`ModelConfig` and `EmbeddingConfig` now contain a new field called `flavour`, which specs whether the model is `tensorflow`, `sklearn`, or `pytorch` model. Struggled to think of a good name for this and am open to suggestions!

Reply:
I vote for `flavorflav` :rofl:?

Reply:
Yeah, boyeee!